lib/semantic_range.rb

```diff
@@ -153,6 +153,12 @@ def self.satisfies?(version, range, loose = false, platform = nil)
     Range.new(range, loose, platform).test(version)
   end
 
+  def self.only_satisfying(versions, range, loose = false, platform = nil)
```

Comment:
maybe a better name? 
`satisfying_subset`
`filter_satisfying`

or just delete this and only do it in our code, but I figure if I want to do it, someone else might too